lib/check.js

```diff
@@ -0,0 +1,13 @@
+
+const checkStatusCode = (status) => {
```

Comment:
I don't think this name is good for what it's doing. It seems to me that it is more like `sanitizeStatusCode`

Reply:
makes sense, thanks for this